fe/fe-core/src/main/java/org/apache/doris/analysis/TableRef.java

```diff
@@ -572,7 +580,12 @@ public void analyzeJoin(Analyzer analyzer) throws AnalysisException {
     public void rewriteExprs(ExprRewriter rewriter, Analyzer analyzer)
             throws AnalysisException {
         Preconditions.checkState(isAnalyzed);
-        if (onClause != null) onClause = rewriter.rewrite(onClause, analyzer, ExprRewriter.ClauseType.ON_CLAUSE);
+        if (onClause != null) {
```

Comment:
What was the predicate ```a.k1=b.k1 and 1=2 ``` changed to before this PR?

Reply:
BoolLital(false)